wagtail/core/models/translation.py

```diff
@@ -0,0 +1,331 @@
+import uuid
```

Comment:
Could we name this `i18n` short for Internationalisation? Since these models are mainly to allow us to organise a site by locales, the actual translation part is done by `wagtail-localize` and `wagtail.contrib.simple_translation`

Reply:
Sure, sounds good. That'll also get around the need for an alias when importing django.utils.translation...

Reply:
Updated to use wagtail.core.models.i18n now.